src/common/Logging/LogCommon.h

```diff
@@ -23,13 +23,13 @@
 // EnumUtils: DESCRIBE THIS
 enum LogLevel : uint8
 {
-    LOG_LEVEL_DISABLED,
-    LOG_LEVEL_FATAL,
-    LOG_LEVEL_ERROR,
-    LOG_LEVEL_WARN,
-    LOG_LEVEL_INFO,
-    LOG_LEVEL_DEBUG,
-    LOG_LEVEL_TRACE,
+    LOG_LEVEL_DISABLED = 0,
+    LOG_LEVEL_FATAL = 1,
+    LOG_LEVEL_ERROR = 2,
+    LOG_LEVEL_WARN = 3,
+    LOG_LEVEL_INFO = 4,
+    LOG_LEVEL_DEBUG = 5,
+    LOG_LEVEL_TRACE = 6,
```

Comment:
Why?

Reply:
i figure it would be in good practice in numbering them as they are in their return in the enuminfo_LogCommon.cpp
https://github.com/azerothcore/azerothcore-wotlk/blob/0ffa270c948841f566d03c89033202c7abbaed75/src/common/Logging/enuminfo_LogCommon.cpp#L53-L60
https://github.com/azerothcore/azerothcore-wotlk/blob/0ffa270c948841f566d03c89033202c7abbaed75/src/common/Logging/enuminfo_LogCommon.cpp#L69-L75